spotxBidAdapter2.js

```diff
@@ -0,0 +1,362 @@
+import * as utils from 'src/utils';
```

Comment:
Review the use of let, const, and var in the file. They all seem to be used and not exactly in the right context all the time.